examples/inverse/plot_lcmv_beamformer.py

```diff
@@ -69,8 +69,10 @@
 colors = ['b', 'k', 'r']
 
 for pick_ori, name, desc, color in zip(pick_oris, names, descriptions, colors):
+    # compute unit-noise-gain beamformer with whitening of the leadfield and
+    # data (enabled by passing a noise covariance matrix)
     stc = lcmv(evoked, forward, noise_cov, data_cov, reg=0.05,
-               pick_ori=pick_ori)
+               pick_ori=pick_ori, weight_norm='unit-noise-gain')
```

Comment:
the output of this script now looks different from: https://martinos.org/mne/dev/auto_examples/inverse/plot_lcmv_beamformer.html

is this the bug with 'max-power' using 'unit-noise-gain'?

btw it's a shame the the doc does not render all the options.